modules/ROOT/pages/queries-aggregations/aggregations.adoc

```diff
@@ -144,10 +137,8 @@ query {
 }
 ----
 
-=== Undirected aggregation queries
-
 When performing an aggregation on related nodes, the query against the relationship
-can be defined as an xref::queries-aggregations/queries.adoc#_undirected_queries[undirected] using the argument `directed: false`:
+can be defined as "undirect" by using the argument `directed: false`:
```

Comment:
I think as an adjective it would make more sense to use "undirected", as we would otherwise call it "directed" rather than "direct".